src/midend/mir_emitter.cpp

```diff
@@ -895,6 +924,64 @@ namespace graphit {
         ctx->addElementType(element_type);
     }
 
+    std::string MIREmitter::convertToFunctorArg(mir::Expr::Ptr arg) {
```

Comment:
Add some comments, especially given there are some restrictions on what Expr type this arg can be.

Reply:
will do.